crates/tx5-connection/src/webrtc/go_pion.rs

```diff
@@ -125,6 +148,29 @@ async fn task(
             while let Some(evt) = peer_evt.recv().await {
                 match evt {
                     Evt::Error(_) => break,
```

Comment:
I'm noticing that this `Error` event only breaks the loop while the states `Disconnected`, `Closed` and `Failed` send an error through the channel. I wonder whether they should be treated equally? But I have to admit that I don't understand the surrounding overall logic yet.